### PR TITLE
chore(parental-leave): Require employer reviewer to insert company national registry id

### PR DESCRIPTION
**Pull request by baering**
## What

Instead of using the national registry id of the reviewer as employer national registry id we ask the reviewer to insert the company's national registry id

## Why

So that the national registry id of the employer is correct

## Screenshots / Gifs

image

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/application/templates/parental-leave/src/lib/dataSchema.ts [hunk 1]
```diff
@@ -51,7 +51,11 @@ export const dataSchema = z.object({
   }),
   shareInformationWithOtherParent: z.enum([YES, NO]),
   usePrivatePensionFund: z.enum([YES, NO]),
-  employerInformation: z.object({ email: z.string().email() }).optional(),
+  employerNationalRegistryId: z
+    .string()
+    .refine((n) => n && kennitala.isValid(n) && kennitala.isPerson(n), {
+      params: errorMessages.otherParentId,
```
Comment (baering) [inline libs/application/templates/parental-leave/src/lib/dataSchema.ts:57]: This message contains a generic `invalid kennitala`. I was wondering if I should create a duplicate to use here but went with the other one.

File: libs/application/templates/parental-leave/src/lib/dataSchema.ts [hunk 2]
```diff
@@ -51,7 +51,11 @@ export const dataSchema = z.object({
   }),
   shareInformationWithOtherParent: z.enum([YES, NO]),
   usePrivatePensionFund: z.enum([YES, NO]),
-  employerInformation: z.object({ email: z.string().email() }).optional(),
```
Comment (baering) [inline libs/application/templates/parental-leave/src/lib/dataSchema.ts:54]: This was an old unused remnant that I decided to remove

File: libs/application/templates/parental-leave/src/lib/dataSchema.ts (hunk 1 above)
Comment (baering) [reply libs/application/templates/parental-leave/src/lib/dataSchema.ts:57]: I'll create a duplicate just in case.

**Review by jeremybarbet — APPROVED** [review]
LGTM 👍

**Pull request merged**